### DISCUSSION TITLE
New text to speech language

### DISCUSSION THREAD

**Discussion by 420755** [community/community#45299] [Other Feature Feedback, Questions, & Ideas]
Hi,
I who'd like to develop a new language.
Are that a big project to develop new language? is available a module for that?
 Is anyone looking to do that for me, or help me to set up the project?
Please let me know.

Thanks.

**Comment by jge162** [comment]
Here you go. https://docs.github.com/en/get-started
this should help you learn the platform. Welcome 🤗

**Comment by 420755** [comment]
Hello will this help me develop a new language from scratch?Sent by Virtual-Mail

**Reply by jge162** [reply]
Can’t help u sorry.

**Comment by anni-anni** [comment]
I'd suggest you find public repositories of already existing [AI text to speech](https://wavel.ai/solutions/realistic-text-to-speech-online/) models. It will give you a better idea of how to construct it rest you will find everything on YouTube nowadays to help you figure out the basic at least...
